modules/installing-oadp-aws-sts.adoc

```diff
@@ -0,0 +1,260 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-aws-sts/oadp-aws-sts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-oadp-aws-sts_{context}"]
+= Installing the OADP Operator and providing the IAM role
+
+AWS Security Token Service (AWS STS) is a global web service that provides short-term credentials for IAM or federated users. This document describes how to install {oadp-first} on an {aws-short} {sts-short} cluster manually.
+
+[IMPORTANT]
+====
+Restic and Kopia are not supported in the OADP {aws-short} {sts-short} environment. Verify that the Restic and Kopia node agent is disabled.
+For backing up volumes, OADP on {aws-short} {sts-short} supports only native snapshots and Container Storage Interface (CSI) snapshots.
+====
+
+[IMPORTANT]
+====
+In an Amazon {aws-short} cluster that uses STS authentication, restoring backed-up data in a different {aws-short} region is not supported.
+
+The Data Mover feature is not currently supported in {aws-short} {sts-short} clusters. You can use native {aws-short} S3 tools for moving data.
+====
+
```

Comment:
Is there some way to separate these admonitions so they are not stacked on top of each other? Maybe move one of them above the first sentence that starts with "AWS Security Token Service (AW STS) is a global web service..."

See: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions "Keep admonitions to a minimum, and avoid placing multiple admonitions close to one another."

Reply:
@gwynnemonahan 
Thank you for the suggestion. Placing the admonition above the opening paragraph doesn't look right. Instead I put the content of both admonitions into one admonition, since it is similar.